lib/opencensus/tags.rb

```diff
@@ -13,14 +15,49 @@
 # limitations under the License.
 
 
+require "opencensus/tags/config"
+require "opencensus/tags/tag_map"
+require "opencensus/tags/formatters"
+
 module OpenCensus
   ##
   # The Tags module contains support for OpenCensus tags. Tags are key-value
   # pairs. Tags provide additional cardinality to the OpenCensus instrumentation
   # data.
   #
-  # TODO: implement tags
-  #
   module Tags
+    ##
+    # Internal key for storing the current TagMap in the thread local
+    # Context
+    #
+    # @private
+    #
+    TAGS_CONTEXT_KEY = :__tags_context__
+
+    class << self
+      ##
+      # Sets the current thread-local TagMap, which used in Stats data
+      # recording.
+      #
+      # @param [TagMap] context
+      #
+      def tags_context= context
```

Comment:
Again, if you're returning a `TagMap`, please name these methods accordingly.

Reply:
Rename method to `tag_map_context`